### DISCUSSION TITLE
Which unlocker to choose for Workstation Pro 17 ?

### DISCUSSION THREAD

**Discussion by gh-SAngeli** [paolo-projects/auto-unlocker#108] [Q&A]
Hi,
I have been using Unlocker for macOS for many years from insanelymac.com website
I thought that this was the official site to consult. I noticed that latest version covers up to Workstation 15

Thanks to references on the Internet I learned that "auto-unlocker" is the official Unlocker for macOS.
I had problems with Ventura installing VMware Tools. I had to give up as it never reboots the Guest OS

Searching on the Internet I cam accross "DrDonk unlocker".
Difference that I noticed is that "auto-unlocker - Unlocker 3" is designed for VMware Workstation 11-15 as the website states.
Rather, "DrDonk Unlocker 4" is designed for VMware Workstation 16/17 as the website states

**Q1. As I might be confused, what version/site should I use?**
I have a purchased copy of VMware Workstation 17 Pro

**Q2.** "hen I might encounter installation issues, like the one I had as above described, where can I ask for help?
**Is there a reliable forum to use, when in need of help?**

Thank you,
Spiro

**Comment by mkitchingh** [comment]
The project has been forked several times. There isn't one single forum that I'm aware of that discusses them. I work one one of the forks (not this one). This fork is active and works well. I would ask here if you need help.

**Reply by gh-SAngeli** [reply]
I really thank for your reply. So far, at present time I wish to understand clearly which unlocker I should use.
To me, reading the description of each one (paolo-projects and DrDonk) DrDonk seems to be more updated for Workstation 17 pro but it might not be the case as paolo-project could be identical to DrDonk. If so, should this project be updated in the description so that end users, lime me, see that even paolo-projecs is compatible with Workstation 17 Pro?

Please can you help me undestanding?

**Reply by mkitchingh** [reply]
That is a choice you would have to make. The versions don't always get updated in the documentation. If it is being updated recently, it is likely fine. This one works fine. This one works fine too. https://github.com/BDisp/unlocker
It's like saying which version of Linux is correct to use. There is more than one option and you just have to pick one. 
They are all typically based on the same original code.

**Comment by gh-SAngeli** [comment]
As for the problem I encountered, using paolo-projects unloker was:
1. very slow performance (very slow os to load) prior to installinv VMware Tools
2. impossible to install VMware Tools. After I see compledes install, when I click on Reboot the system try to reboot but then hangs all the time. I have to restart the Goest Machine and this would end up not having installed Toos. So, using Ventura without VMWare Tools ends up havng a very slow OS to manage/use

**Comment by robertapengelly** [comment]
@gh-SAngeli Sorry for the delay I didn't see the email about the messages.  Anyways, I'm a little confused as to what your using.  Are you using this auto-unlocker? If you are is there a reason your not using https://github.com/paolo-projects/unlocker instead?  I have both Monterey and Ventura installed with VMware Workstation 17 and both work without problems.

**Reply by KINGLIFER** [reply]
20 is out and I am scared to upgrade!!!! lol
